### PR TITLE
added debugger in switch-reset module

### PR DESCRIPTION
**Pull request by gaurav-calsoft**

### CODE REVIEW

File: ansible/library/pn_switch_config_reset.py [hunk 1]
```diff
@@ -147,16 +147,32 @@ def main():
...
     if reset_ips:
-        # Wait 120 secs for nvOS to come up
-        time.sleep(120)
+        # Wait 180 secs for nvOS to come up
+        time.sleep(180)
 
         # Check until we are able to ssh into switches
         for ip in reset_ips:
-            while True:
+            epocs = 0
```
Comment (ramanujjaju) [inline ansible/library/pn_switch_config_reset.py]: What is epocs variable for and why is it 6? Can you please include a comment here?

Comment (gaurav-calsoft) [inline ansible/library/pn_switch_config_reset.py]: @ramanujjaju Epocs is a iteration variable. Basically there is sleep of 10s in each iteration. So, after 180s of initial sleep after switch-reset, the code will try to do ssh max 6 times with the delay of 10s in each iteration. So, roughly we are waiting for 1 more minute(which includes 6 tries).

**Comment by amitsi** [comment]
what is this debugger do?

**Comment by amitsi** [comment]
does

**Comment by ashxjain** [comment]
@amitsi That was added to root cause an issue seen in switch reset module, which is now fixed

**Pull request merged**